Rules/UseSingularNouns.cs

```diff
@@ -57,15 +72,18 @@ public class CmdletSingularNoun : IScriptRule {
                     // Convert the noun part of the function into a series of space delimited words
                     // This helps the PluralizationService to provide an accurate determination about the plurality of the string
                     nounPart = SplitCamelCaseString(nounPart);
-                    var words = nounPart.Split(new char [] { ' ' });
+                    var words = nounPart.Split(new char[] { ' ' });
```

Comment:
We should elevate this into a static readonly at some point

Reply:
Unless there's an empty overload that will do that for us???

Reply:
I ended up rewriting this to not create a bunch of strings or use regex -- now we just search backward for the index of a case change and split there